docs/console-api/daily-reports.mdx

```diff
@@ -57,50 +57,4 @@ import Rapidoc from "../../src/components/Rapidoc"
 
 - New columns may be added in the future.
 
-<br/>
```

Comment:
this should go above, but when I was looking at this docs page, I got confused.  I wasn't sure what the endpoint was to hit.  is it `statsigapi.net/v1/daily_reports`? Or `/daily_reports/first_exposures`?  Make it more clear at the beginning what the endpoint path is - the rapidoc example didnt have this anywhere I could see

Reply:
I think an improve like this could help all the rapid-doc endpoints. This should be the first thing you see